sources/Malbim/Jonah/Malbim_Jonah_parse_text.py

```diff
@@ -0,0 +1,126 @@
+# -*- coding: utf-8 -*-
+import sys
+import os
+# for a script located two directories below this file
+p = os.path.dirname(os.path.dirname(os.path.dirname(os.path.abspath(__file__))))
+sys.path.insert(0, p)
+sys.path.append("/Users/Eytan/Desktop/sefaria/Sefaria-Data/sources")
```

Comment:
This line can't be here if I'm going to try to run this script from my system. The correct way to import a module is:
`import sources`